src/main/java/core/basesyntax/BallColors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum BallColors {
+    Yellow,
+    Green,
+    Red,
+    Purple,
+    White,
+    Black
```

Comment:
Enum names should be in uppercase letters. According to the Java style guide, the names of enum values should be in uppercase letters. For example, `YELLOW` instead of `Yellow`.